roles/kubernetes/preinstall/defaults/main.yml

```diff
@@ -7,7 +7,6 @@ ignore_assert_errors: false
 epel_enabled: false
 
 common_required_pkgs:
-  - python-httplib2
```

Comment:
I don't know if `python-httplib2` is effectively not required on SLES but, if not and if it's removed from here, it should be added in vars files in roles/kubernetes/preinstall/vars/ for all other distros

Reply:
btw, I have this installed in bootstrap in https://github.com/kubernetes-incubator/kubespray/blob/3f37e1c4088bd62a8f93c5c393f082f4ad5ca7d1/roles/bootstrap-os/tasks/main.yml#L83 via my fix for https://github.com/kubernetes-incubator/kubespray/issues/1032